app/src/main/java/com/cornellappdev/android/eatery/ui/screens/NotificationsHomeScreen.kt

```diff
@@ -0,0 +1,66 @@
+package com.cornellappdev.android.eatery.ui.screens
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.android.eatery.ui.components.notifications.FavoriteItemRow
+import com.cornellappdev.android.eatery.ui.theme.EateryBlue
+import com.cornellappdev.android.eatery.ui.theme.EateryBlueTypography
+
+@Composable
+fun NotificationsHomeScreen(
+
+) {
```

Comment:
Ah ok so this is just UI for this screen, no ViewModel. Add a TODO to add a hilt viewmodel here :-)